snippets/sdp/gpio/icbmsg/flpr.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SDP_GPIO_BACKEND_ICBMSG=y
```

Comment:
```suggestion
CONFIG_SDP_GPIO_BACKEND_ICBMSG=y
CONFIG_IPC_SERVICE=y
```
Add it here or general in application Kconfig?

Reply:
Added to application Kconfig